core/arch/arm/include/mm/tee_mmu_defs.h

```diff
@@ -69,7 +69,7 @@
 #define TEE_MMU_L1_ALIGNMENT		TEE_MMU_L1_SIZE
 
 #define TEE_MMU_L2_NUM_ENTRIES		(TEE_MMU_L2_SIZE / 4)
-#define TEE_MMU_L2_SIZE			(1 << 10)
```

Comment:
This size is supposed to be the size of a single page table.
Please use a method similar to the one in core_mmu_lpae.c (MAX_XLAT_TABLES).
All these defines could probably be moved into core_mmu_v7.c